src/types/index.ts

```diff
@@ -149,7 +149,11 @@ export type PaymentStatus =
   | 'processing'
   | 'waitingForInput';
 
-export type PaymentMethodType = 'cardPresent' | 'interacPresent' | 'card';
+export type PaymentMethodType =
```

Comment:
i think we also need `requiresAction` for PaymentIntent.Status here https://github.com/stripe/stripe-terminal-react-native/blob/52eb078f0f3d3e4ceda60456b1be0c431e4b6c33/src/types/PaymentIntent.ts#L23